sdks/java/io/jdbc/src/main/java/org/apache/beam/sdk/io/jdbc/JdbcIO.java

```diff
@@ -1694,6 +1700,7 @@ private void executeBatch() throws SQLException, IOException, InterruptedExcepti
         if (records.isEmpty()) {
           return;
         }
+        Long startTimeNs = System.nanoTime();
```

Comment:
Any reason to choose `System.nanoTime()` vs `System.currentTimeMillis()`?

Reply:
`currentTimeMillis` may be affected by clock adjustments in the machine, while `nanoTime` measures ellapsed time in the machine, so it will not be affected by that.

Reply:
https://stackoverflow.com/questions/1770010/how-do-i-measure-time-elapsed-in-java

Reply:
Thanks, LGTM.